proto/one/operations/spreadsheet/historian_binding.proto

```diff
@@ -3,7 +3,17 @@ syntax = "proto3";
 option csharp_namespace ="ONE.Models.CSharp";
 
 message HistorianBinding {
-    int32 startTime = 2;   // Inclusive, base on operation timezone and time unit is in minutes.
-    int32 stopTime = 3;    // Exclusive, base on operation timezone and time unit is in minutes.
+
+    // Inclusive, base on operation timezone and time unit is in minutes.
+    int32 startTime = 2;   
```

Comment:
Please do not change it but anyone know why we started numbering at 2?

Reply:
I think there may have  been a field at 1 previously